lib/Model.php

```diff
@@ -571,7 +572,7 @@ public function attributes()
 	/**
 	 * Retrieve the primary key name.
 	 *
-	 * @param boolean Set to true to return the first value in the pk array only
+	 * @param boolean $first Set to true to return the first value in the pk array only
```

Comment:
> I didnt modify existing docblocks for grammar or style.

https://github.com/robinpowered/php-activerecord/pull/4/files#diff-c2ede32dd201390a146144c5df4b541cR80

I don't really care though, really would like to just move forward.